modules/user_manual/pages/files/access_webdav.adoc

```diff
@@ -287,21 +285,20 @@ To map the drive:
 computer Z drive:
 +
 ....
-net use Z: https://<drive_path>/remote.php/dav/files/USERNAME/ /user:youruser
-yourpassword
+net use Z: https://<drive_path>/remote.php/dav/files/USERNAME/ /user:youruser yourpassword
 ....
 
-______________________________________________________
+""
 where <drive_path> is the URL to your ownCloud server.
-______________________________________________________
+""
 
 For example:
-`net use Z: https://example.com/owncloud/remote.php/dav/files/USERNAME/  /user:youruser yourpassword`
 
-___________________________________________________________________________
-The computer maps the files of your ownCloud account to the drive letter
-Z.
-___________________________________________________________________________
+`net use Z: https://example.com/owncloud/remote.php/dav/files/USERNAME/ /user:youruser yourpassword`
+
+""
```

Comment:
This isn't a quote. It'd be better as an admonition, or as part of the sentence preceding the command.